infrastructure/src/main/java/org/corfudb/infrastructure/management/ClusterStateContext.java

```diff
@@ -23,7 +23,6 @@
     /**
      * Current cluster state, updated by failure detector on current iteration.
      */
-    @Getter
```

Comment:
The corresponding import for Getter is unused now.

Reply:
done